product_docs/docs/edbcloud/beta/getting_started/01_connect_cloud_account.mdx

```diff
@@ -4,15 +4,13 @@ title: "Connecting Your Cloud Account"
 
 This topic describes how to set up your EDB Cloud subscription on Azure Marketplace. Your Azure subscription is where EDB Cloud can create and manage PostgreSQL clusters.
```

Comment:
I would change the first sentence to: "This topic describes how to set up your EDB Cloud account through the Microsoft Azure Marketplace."